controlplane/pkg/nsmd/nsmd.go

```diff
@@ -420,7 +422,30 @@ func StartAPIServerAt(server NSMServer, sock net.Listener) error {
 	return nil
 }
 
-func GetExcludedPrefixes(serviceRegistry serviceregistry.ServiceRegistry) ([]string, error) {
+func GetExcludedPrefixes(serviceRegistry serviceregistry.ServiceRegistry) (prefix_pool.PrefixPool, error) {
+	excludedPrefixes := []string{}
+
+	excludedPrefixesEnv, ok := os.LookupEnv(ExcludedPrefixesEnv)
+	if ok {
+		logrus.Infof("Get excludedPrefixes from ENV: %v", excludedPrefixesEnv)
+		excludedPrefixes = append(excludedPrefixes, strings.Split(excludedPrefixesEnv, ",")...)
+	}
+
+	configMapPrefixes, err := getExcludedPrefixesFromConfigMap(serviceRegistry)
+	if err != nil {
+		logrus.Warnf("Cluster is not support kubeadm-config configmap, please specify PodCIDR and ServiceCIDR in %v env", ExcludedPrefixesEnv)
```

Comment:
Should we point to documentation of howto do so?

Reply:
It's temporary message, until situation with getting PodCIDR and ServiceCIDR in general case becomes clear